cfgov/v1/atomic_elements/organisms.py

```diff
@@ -696,9 +710,14 @@ class Expandable(BaseExpandable):
 class ExpandableGroup(blocks.StructBlock):
     heading = blocks.CharBlock(required=False)
     body = blocks.RichTextBlock(required=False)
-
     is_accordion = blocks.BooleanBlock(required=False)
-    has_rule = blocks.BooleanBlock(required=False)
+    has_rule = blocks.BooleanBlock(
+        default=False,
+        required=False,
+        label="Has top rule line",
+        help_text=('Check this to add a horizontal rule line to top of '
+                   'expandable group.')
+    )
```

Comment:
As it turns out, due to a series of unfortunate events, this field actually triggers a rule line both above _and_ below the ExpandableGroup module. Without getting into the aforementioned data migration issue, the fix here is straightforward, but hard to explain in a comment here. Let's pair on it tomorrow or Thursday.

Reply:
That is unfortunate. :) What are you thinking for the fix? I checked the live instances of ExpandableGroup on cf.gov (there aren't too many) and it looks like we only use the 'has_rule' option [on one page](https://www.consumerfinance.gov/about-us/events/request-speaker/). I'm wondering, could we replace `has_rule` with `has_top_rule_line` here and then update the affected page once the change is deployed?

Reply:
Oh! I didn't even think to see how much it was being used. That's a much better idea than the one I had, which was to add more logic to `render_block.html`.

_(Also, I bet if the rule line on that page just went away, no one would notice 😁 …)_

Reply:
@Scotchester That is probably true! 😄 

I updated `has_rule` to `has_top_rule_line` on ExpandableGroup in https://github.com/cfpb/cfgov-refresh/pull/3476/commits/a67606e1c8d308c1f872928868ef0b08a2e08e6b. Thanks for noticing this issue!